app/_src/guides/federate.md

```diff
@@ -0,0 +1,139 @@
+```
+
+We skip default mesh creation as we will bring mesh from zone control plane in the next steps.
+
+### Sync endpoint
+
+Find the external IP and port of the `{{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync` service in the `{{site.mesh_namespace}}` namespace:
+
+```sh
+kubectl get services -n {{site.mesh_namespace}}
+NAMESPACE     NAME                   TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                                                                  AGE
+{{site.mesh_namespace}}   {{site.mesh_cp_zone_sync_name_prefix}}global-zone-sync     LoadBalancer   10.105.9.10     35.226.196.103   5685:30685/TCP                                                           89s
+{{site.mesh_namespace}}   {{site.mesh_cp_name}}     ClusterIP      10.105.12.133   <none>           5681/TCP,443/TCP,5676/TCP,5677/TCP,5678/TCP,5679/TCP,5682/TCP,5653/UDP   90s
+```
+
+In this example the value is `35.226.196.103:5685`. You pass this as the value of `<global-kds-address>` when you federate the zone control plane.
+If you see `<Pending>` you either need to wait until load balancer is provisioned or you need to make sure that your Kubernetes cluster support provisioning load balancers. 
+
+## Copy resources from zone to global control plane
+
+To federate zone control plane without any traffic interruption, we need to copy resources like secrets, meshes etc.
+We can do this by executing the following command:
+
+```sh
+kubectl --context=mesh-zone port-forward svc/demo-app -n kuma-demo 5000:5000
+export ZONE_USER_ADMIN_TOKEN=$(kubectl --context=mesh-zone get secrets -n kuma-system admin-user-token -ojson | jq -r .data.value | base64 -d)
+kumactl config control-planes add \
+  --address http://localhost:5681 \
+  --headers "authorization=Bearer $ADMIN_TOKEN" \
+  --name "zone-cp" \
+  --overwrite  
+  
+kumactl export --profile=federation --format=kubernetes > resources.yaml
+```
+
+Apply resources on global control plane
+```sh
+kubectl apply --context=mesh-global -f resources.yaml
+```
+
+## Connect zone control plane to global control plane
+
+Update Helm deployment of zone control plane to configure connection to the global control plane.
+
+```sh
+helm upgrade --kube-context=mesh-zone --namespace {{site.mesh_namespace}} \
+--set {{site.set_flag_values_prefix}}controlPlane.mode=zone \
+--set {{site.set_flag_values_prefix}}controlPlane.zone=zone-1 \
+--set {{site.set_flag_values_prefix}}ingress.enabled=true
+--set {{site.set_flag_values_prefix}}controlPlane.kdsGlobalAddress=grpcs://<global-kds-address>:5685 \
+--set {{site.set_flag_values_prefix}}controlPlane.tls.kdsZoneClient.skipVerify=true \
+{{ site.mesh_helm_install_name }} {{ site.mesh_helm_repo }}
+```
+
+## Verify federation
+
+To verify federation, first port-forward the API service from the global control plane. 
+
+```sh
+kubectl --context=mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:5681
```

Comment:
nitpick: I'd port forward this to some other port to not make it clash with zone cp port

```suggestion
kubectl --context=mesh-global port-forward svc/{{site.mesh_cp_name}} -n {{site.mesh_namespace}} 5681:15681
```